states/alabama/elections.toml

```diff
@@ -3,7 +3,60 @@ date = 2020-07-14
 name = "Congressional Primary Run-Off (if needed)"
 original_date = 2020-07-14
 
+[20200714.registration]
+received_by = 2020-06-29
```

Comment:
Looks like you can register online and hand deliver the registration forms as well. ( https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action ) Therefore `online_by` and `in_person_by` should be listed as well.

Reply:
Interesting that the primary notes different dates.

<img width="551" alt="Screen Shot 2020-06-27 at 10 16 23 AM" src="https://user-images.githubusercontent.com/52649/85928091-5246d880-b85f-11ea-8300-90ba4ae274d9.png">